libs/newsletter-workflow/src/lib/steps/launchNewsletter/noItem.ts

```diff
@@ -0,0 +1,19 @@
+import type { WizardStepLayout } from '@newsletters-nx/state-machine';
+
+export const noItemLayout: WizardStepLayout = {
+	staticMarkdown: `# Launch a newsletter
+
+This wizard will guide you through the process of launching a new newsletter using email-rendering.
+
+**no draft has been selected to launch**
+
+`,
+	role: 'CREATE_START',
```

Comment:
Need to add a label so doesn't display as "noItem" in the breadcrumbs

Reply:
👍 I'll label it as 'start' and use the same label for the `launchNewsletterLayout` (the initial step for when there is a selected item)